keyboards/cantor/rules.mk

```diff
@@ -0,0 +1,5 @@
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+SPLIT_KEYBOARD = yes
+SERIAL_DRIVER = usart
+VIA_ENABLE      = yes    # Enable VIA
+VIAL_ENABLE      = yes    # Enable VIAL
```

Comment:
this should go inside a keymap

Reply:
Alright I moved those lines into keyboards/cantor/vial/rules.mk